src/Akka.Hosting.API.Tests/verify/CoreApiSpec.ApprovePersistence.verified.txt

```diff
@@ -64,19 +69,23 @@ namespace Akka.Persistence.Hosting
         public string ConnectionString { get; set; }
         public System.TimeSpan ConnectionTimeout { get; set; }
         public abstract string MetadataTableName { get; set; }
+        public abstract System.Data.IsolationLevel ReadIsolationLevel { get; set; }
```

Comment:
Shouldn't this only be in the Sql-specific plugins?

Reply:
Yes, this change is inside `SqlJournalOptions` class

Reply:
Ah - I didn't realize those types were in the base Akka.Persistence.Hosting package.